KroneMultimedia/SharedFastfile.rb

```diff
@@ -0,0 +1,289 @@
+lane :before_build do
+  clear_derived_data
+  increment_build_number({
+      build_number: latest_testflight_build_number+1
+  })
+  cocoapods
+  unlock_keychain
+end
+
+lane :badge do
+  badge(shield: ENV['CI_BUILD_REF_NAME'], no_badge: true)
+end
+desc "Run Unit/UI Tests"
+lane :snapshots do
+  tests
+  #### PLACE FOR AWS
+end
+
+desc "Kill Simulator"
+lane :kill_simulator do
+  sh "killall iOS Simulator &> /dev/null || true"
+end
+
+desc "Run Unit/UI Tests"
+lane :tests do
+  certificates
+
+  kill_simulator
+  snapshot( {
+      workspace: $self_workspace,
+      scheme: $self_scheme,
+      devices: [ "iPhone 6" ],
+      languages: [ "en-US" ],
+      clear_previous_screenshots: true,
+      reinstall_app: true,
+      clean: true,
+      buildlog_path: "fastlane/logs",
+      skip_open_summary: true
+  })
+  sh "cat logs/*|xcpretty --report html --output logs/test_report.html  -t 2>&1 >> /dev/null"
+  kill_simulator
+end
+
+lane :certificates do
+
+  $self_idfs.each do |idf|
+     match(git_url: "ssh://git@certs/mobile/IOS-CERTIFICATES.git", 
+        type: "development", 
+        app_identifier: idf)
+
+     match(git_url: "ssh://git@certs/mobile/IOS-CERTIFICATES.git", 
+        type: "appstore", 
+        app_identifier: idf)
+
+ end
+
+end
+
+lane :build_ipa do
+   before_build
+   certificates
+
+  gym({
```

Comment:
Could you remove all the `{` in the Ruby configuration files and just have `( ... )`?